libpod/container.go

```diff
@@ -133,9 +133,8 @@ type ContainerConfig struct {
 	SharedNamespaceMap map[string]string `json:"sharedNamespaces"`
 	// Time container was created
 	CreatedTime time.Time `json:"createdTime"`
-
-	// TODO save log location here and pass into OCI code
-	// TODO allow overriding of log path
+	// LogPath log location
+	LogPath string `json:"logPath"`
```

Comment:
Going to need database work to get this saved. Might be best to hold off until the database changes already in the networking PR land.